### ISSUE TITLE
Chosen ICONS are ignored 

### ISSUE THREAD

**Issue by BjornS98** [marksie1988/atomic-calendar-revive#178]
After update HA, the icons I have set up for the following calendars are ignored, and the standard (gift,home,star) is used. Somebody got an idea of how this is happening?

```
  - CalEventIcon1: 'mdi:account'
    CalEventIcon2: 'mdi:white-balance-sunny'
    CalEventIcon3: 'mdi:delete-empty'
    defaultMode: 2
    entities:
      - entity: calendar.calendar
        type: icon1
      - entity: calendar.feestdagen_in_belgie
        type: icon2
      - entity: calendar.afval
        type: icon3
    name: Calendar
    showNoEventsForToday: false
    showWeekDay: false
    sortByStartTime: false
    title: Calendar
    type: 'custom:atomic-calendar-revive'
    showColors: true
    showDate: false
```

**Comment by marksie1988** [comment]
Hey,

I'll take a look at this as soon as I get chance. 

I'm currently working on v2 of the card which has quite a lot of changes that I'm working through. So it's taking me a little longer to look at issues.

**Comment by BjornS98** [comment]
Thanks for the quick response! A V2 would be nice! And no worrys the card works fine, just that small issue I was facing this week. 

Keep up the good work!

**Comment by marksie1988** [comment]
Fixed in #237
